src/utils/bounding_box.ts

```diff
@@ -54,10 +56,10 @@ export function boundingBox(elements: SDFGElement[]): DOMRect {
     });
 
     const ret_bb = new DOMRect(
-        bb.x1 ? bb.x1 : 0,
-        bb.y1 ? bb.y1 : 0,
-        (bb.x2 ? bb.x2 : 0) - (bb.x1 ? bb.x1 : 0),
-        (bb.y2 ? bb.y2 : 0) - (bb.y1 ? bb.y1 : 0)
+        (bb.x1 ? bb.x1 : 0) - padding,
```

Comment:
@phschaad I think padding should be in percentage of viewport size, rather than an absolute number. What do you think?

Reply:
That is a good idea, it will probably perform better.